qa/os/src/test/java/org/elasticsearch/packaging/test/PackagesSecurityAutoConfigurationTests.java

```diff
@@ -154,10 +154,10 @@ public void test71ReconfigureFailsWhenKeyStorePasswordWrong() throws Exception {
         verifyPackageInstallation(installation, distribution(), sh);
         verifySecurityAutoConfigured(installation);
         assertNotNull(installation.getElasticPassword());
-        Shell.Result changePassword = installation.executables().keystoreTool.run(
-            "passwd",
-            "some-password" + "\n" + "some-password" + "\n"
-        );
+        Shell.Result changePassword = installation.executables().keystoreTool.run("passwd", """
+            some-password
+            some-password
+            """);
```

Comment:
Given how short the second string is, I personally think this code would be more readable as:
```suggestion
        Shell.Result changePassword = installation.executables().keystoreTool.run("passwd", "some-password\nsome-password\n");
```